webrender/src/render_backend.rs

```diff
@@ -132,6 +136,10 @@ impl RenderBackend {
                                                                    format,
                                                                    bytes);
                         }
+                        ApiMsg::Flush => {
+                            let mut flush_notifier = self.flush_notifier.lock();
+                            flush_notifier.as_mut().unwrap().as_mut().unwrap().all_messages_flushed();
```

Comment:
@changm I see, thanks for the explanation.

> Finally, the flush here is to flush the ApiMsg pipeline.

I understand how you use it and how it works. I'm saying there is no actual "flush" action associated with it, so it might be called something more general. But it doesn't matter since the change is temporary anyway.